components/TextArea/TextArea.jsx

```diff
@@ -192,8 +170,14 @@ TextArea.propTypes = {
     baseFontSize: PropTypes.number,
   }),
   skin: PropTypes.oneOf(['default', 'dark']),
+  innerRef: PropTypes.oneOfType([
+    PropTypes.func,
+    PropTypes.shape({ current: PropTypes.instanceOf(Element) }),
+  ]),
```

Comment:
Not possible as in TextAreaComponent it is not possible to get the ref from properties (in functional components) as it comes only as second argument of forwardRef and it was intentional not to use it there so as not to affect the doc.

In the case of the link component it is correct because the ref used is the second parameter of forwardRef and is passed directly to the styled-components component.

Reply:
Hummm, I get it, so in  this case will be necessary update the typescript definitions for component with the new prop.